#Goal Selection Param File ~#

#~ General Parameters
slam_map: /map                              # SLAM map topic
coverage_map: coverage_map                 # Coverage map topic
map_frame: map                              # Map frame
base_footprint_frame: robot2_tf/base_footprint        # Base footprint frame
sensor_frame: robot2_tf/base_link
dist_of_use: 4
field_of_view: 60

# velocity pub topic
velocity_pub: /robot2/mobile_base/commands/velocity
# resolution used to be 0.05
# change also delta param according in
# gmapping.launch.xml in turtlebot
# navigation package
resolution: 0.04          # Map resolution

# Radius around pose
radius: 2

# Size Of Coverage Map
#width: 704
#height: 576
##height: 480
#
width: 864
height: 896
#height: 480

# Map origin
#origin_x: -29.8
#origin_y: -29.8
#origin_y: -20.2
origin_x: -16.36
origin_y: -18.92
#origin_y: -20.2



  
